RecoTracker/LSTCore/interface/Circle.h

```diff
@@ -0,0 +1,43 @@
+#ifndef RecoTracker_LSTCore_interface_Circle_h
+#define RecoTracker_LSTCore_interface_Circle_h
+
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+
+namespace lst {
+
+  template <typename TAcc>
+  ALPAKA_FN_ACC ALPAKA_FN_INLINE float computeRadiusFromThreeAnchorHits(
+      TAcc const& acc, float x1, float y1, float x2, float y2, float x3, float y3, float& g, float& f) {
```

Comment:
Picky comment but since Matti also commented on this lines and you may be changing it:
At some point we were asked (https://github.com/cms-sw/cmssw/pull/45117#discussion_r1844301482) to change those `g`s and `f`s to more meaningful names, like `regressionCenterX` and `regressionCenterY`, which we did in a follow-up PR. We could consider following that convention also here?
(Sorry for missing this before)